### ISSUE TITLE
WaveRNN hard to generate good quality

### ISSUE THREAD

**Issue by OswaldoBornemann** [mozilla/TTS#674]
I have tried to train wavernn, but it seems that hard to generate good quality mel spec. Hope someone could give me a hint. 
![Screen Shot 2021-03-02 at 4 09 38 PM](https://user-images.githubusercontent.com/11624203/109617766-c31e5d00-7b71-11eb-8cf0-81655996fe69.png)
![Screen Shot 2021-03-02 at 4 09 33 PM](https://user-images.githubusercontent.com/11624203/109617778-c74a7a80-7b71-11eb-9ff4-9175c349344c.png)

**Comment by erogol** [comment]
what configuration do you use?

**Comment by OswaldoBornemann** [comment]
@erogol I used the default configuration under `vocoder/configs/wavernn_config.json`. I haven't changed any parameters but just the output path and dataset.

**Comment by OswaldoBornemann** [comment]
I have changed the learning rate to 1e-4, now the result seems better.

But also i have two main questions.
The first problem (the first part of the circle) is the spectrum trained by the model. The position of the horizontal line (formant position) seems to be curved, but the position of the original spectrum is straight.
The second problem (the second part circled) is that I found that the model will have extra noise in the synthesis of some silent sections or divided sections, but the position of the original audio is relatively clear.

![3071614849618_ pic_hd](https://user-images.githubusercontent.com/11624203/109944328-17137800-7d11-11eb-97f2-8fba985901d6.jpg)
